app/views/devise/registrations/edit.html.erb

```diff
@@ -1,6 +1,6 @@
 <h2>Edit <%= resource_name.to_s.humanize %></h2>
 
-<%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f| %>
+<%= form_for(@user, as: :user, url: registration_path(@user), html: { method: :put }) do |f| %>
```

Comment:
I'm not sure I understand the purpose of switching this from the generalized version to be specific only to the `User` model.